packages/bot-web-ui/src/components/summary/summary.scss

```diff
@@ -1,5 +1,8 @@
 .summary {
+    display: flex;
     padding: 16px;
```

Comment:
I know it is not done by you and it is not part of the requirement but can you please
replace 16px with 1.6rem? @maryia-matskevich-deriv

Reply:
@sandeep-deriv done ✅